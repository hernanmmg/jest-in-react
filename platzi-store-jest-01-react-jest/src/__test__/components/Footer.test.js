import React from 'react';
import { mount } from 'enzyme';
import { create } from 'react-test-renderer';
import Footer from '../../components/Footer';
import 'jsdom-global/register';

describe('<Footer />', () => {
  const footer = mount(<Footer />);
  test('renders del componente Footer', () => {
    expect(footer.length).toEqual(1);
  });

  test('renders title', () => {
    expect(footer.find('.Footer-title').text()).toEqual('Platzi Store');
  });

  test('checks Footer component s UI', () => {
    const footer = create(<Footer />);
    expect(footer.toJSON).toMatchSnapshot();
  });
});
