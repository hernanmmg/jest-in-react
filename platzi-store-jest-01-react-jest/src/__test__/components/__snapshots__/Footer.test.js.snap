// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> checks Footer component s UI 1`] = `[Function]`;
